<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-07-31T12:38:52Z" name="Default suite" finished-at="2022-07-31T12:39:02Z" duration-ms="10105">
    <groups>
    </groups>
    <test started-at="2022-07-31T12:38:52Z" name="Default test" finished-at="2022-07-31T12:39:02Z" duration-ms="10105">
      <class name="com.stringdemo.practice.StringEqualsTest">
        <test-method signature="stringEqualTest()[pri:0, instance:com.stringdemo.practice.StringEqualsTest@12d2ce03]" started-at="2022-07-31T18:08:52Z" name="stringEqualTest" finished-at="2022-07-31T18:09:02Z" duration-ms="10080" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [hello] but found [jay ho]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [hello] but found [jay ho]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at org.testng.asserts.Assertion$11.doAssert(Assertion.java:201)
	at org.testng.asserts.Assertion.executeAssert(Assertion.java:30)
	at org.testng.asserts.Assertion.doAssert(Assertion.java:15)
	at org.testng.asserts.Assertion.assertEquals(Assertion.java:198)
	at com.stringdemo.practice.StringEqualsTest.stringEqualTest(StringEqualsTest.java:16)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stringEqualTest -->
      </class> <!-- com.stringdemo.practice.StringEqualsTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
